extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js

```diff
@@ -67,6 +73,63 @@ describes.realWin('amp-jwplayer', {
           'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html');
     });
   });
+  it('renders with a playlist and parses contextual parameter', () => {
```

Comment:
@boyntoni One more thing, if you'd like, you can flatten the structure of these tests with `async/await`.

```js
  it('renders with a playlist and parses contextual parameter', async() => {
    const jw = await getjwplayer({
      'data-playlist-id': '482jsTAr',
      'data-player-id': 'sDZEo0ea',
      'data-content-search': '__CONTEXTUAL__',
      'data-content-contextual': true,
    });
    const iframe = jw.querySelector('iframe');
    expect(iframe).to.not.be.null;
    expect(iframe.tagName).to.equal('IFRAME');
    expect(iframe.src).to.equal(
        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=title_tag&contextual=true');
    });
```